Formula/b/berkeley-db.rb

```diff
@@ -5,7 +5,7 @@ class BerkeleyDb < Formula
   mirror "https://fossies.org/linux/misc/db-18.1.40.tar.gz"
   sha256 "0cecb2ef0c67b166de93732769abdeba0555086d51de1090df325e18ee8da9c8"
   license "AGPL-3.0-only"
-  revision 2
+  revision 3
```

Comment:
Doesn't seem like this feature is interesting enough for most users to make it worth forcing everyone to reinstall `berkeley-db` just to pick up this change:
```suggestion
  revision 2
```

Reply:
Sure!  I've made the change, re-squashed & rebased just now! 🎉